packages/contracts-bedrock/src/L2/IOptimismSuperchainERC20.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+/// @title IOptimismSuperchainERC20
+/// @notice This interface is available on the OptimismSuperchainERC20 contract.
+///         We declare it as a separate interface so that it can be used in
+///         custom implementations of SuperchainERC20.
+interface IOptimismSuperchainERC20 {
+    /// @notice Emitted whenever tokens are minted for an account.
+    /// @param account Address of the account tokens are being minted for.
+    /// @param amount  Amount of tokens minted.
+    event Mint(address indexed account, uint256 amount);
+
+    /// @notice Emitted whenever tokens are burned from an account.
+    /// @param account Address of the account tokens are being burned from.
+    /// @param amount  Amount of tokens burned.
+    event Burn(address indexed account, uint256 amount);
+
+    /// @notice Emitted whenever tokens are sent to another chain.
+    /// @param from         Address of the sender.
+    /// @param to           Address of the recipient.
+    /// @param amount       Amount of tokens sent.
+    /// @param destination  Chain ID of the destination chain.
+    event SentERC20(address indexed from, address indexed to, uint256 amount, uint256 destination);
```

Comment:
We will want to make sure this interface matches the [spec](https://github.com/ethereum-optimism/specs/blob/main/specs/interop/token-bridging.md#senderc20-1) exactly, its `SendERC20` rather than `SentERC20`

Reply:
@tynes I pushed this change but @agusduha mentioned he wrote them in past tense to match the events on current contracts like [CrossDomainMessenger](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol#L163) or [L2ToL2CrossDomainMessenger](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol#L137). I think this may be a good change for consistency between contracts, I can push it to the spec if you agree.